queries.sql

```diff
@@ -80,4 +80,28 @@ SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1
 GROUP BY species;
  species |        avg         
 ---------+--------------------
- pokemon | 3.0000000000000000
\ No newline at end of file
+ pokemon | 3.0000000000000000
+
+
+
+--  MULTIPLE TABLES DAY 3 
+-- What animals belong to Melody Pond?
+SELECT name, full_name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE full_name = 'Melody Pond';
+
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT animals.name, species.name FROM animals JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT full_name, name  FROM animals RIGHT JOIN owners ON animals.owner_id = owners.id;
+
+-- How many animals are there per species?
+SELECT species.name, COUNT(*) FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.name;
+
+-- List all Digimon owned by Jennifer Orwell.
+SELECT animals.name, species.name, full_name FROM animals JOIN species ON animals.species_id = species.id JOIN owners ON animals.owner_id = owners.id WHERE species.name = 'Digimon' AND full_name = 'Jennifer Orwell';
```

Comment:
- Please fix the result for this query after fixing the issue related to the species assigned, mentioned above. 👍 👍